Libraries/Components/ScrollResponder.js

```diff
@@ -113,7 +114,16 @@ type State = {
   observedScrollSinceBecomingResponder: boolean,
   becameResponderWhileAnimating: boolean,
 };
-type Event = Object;
+type Event = $ReadOnly<{|
```

Comment:
Is `SyntheticEvent` from `CoreEventTypes` helpful here?

Also, are we sure this shouldn't be `ScrollEvent` from `CoreEventTypes`? Where do you see that this event contains touches?

Reply:
In response to your review, I changed PR a lot.

I found that the event defined by the existing flow type is getting passed in the arguments of all the functions defined in `ScrollResponder`, so we don't need to use `SysntheticEvent` here.